benchmark/README.md

```diff
@@ -0,0 +1,64 @@
+# Duck-Read-Cache-FS Performance Benchmark Suite
+
+## Overview
+This benchmark suite evaluates the performance of our DuckDB fs extension compared to the standard DuckDB httpfs. 
+
+## Configuration
+
+### AWS Credentials
+Set up your AWS credentials in your environment:
+```bash
+export AWS_ACCESS_KEY_ID='your-key-id'
+export AWS_SECRET_ACCESS_KEY='your-secret-key'
+export AWS_DEFAULT_REGION='your-region'
+```
+
+### Available Benchmark Suites
+Available benchmark suites:
+```bash
+build/release/extension/read_cache_fs/read_s3_object
+build/release/extension/read_cache_fs/sequential_read_benchmark
+build/release/extension/read_cache_fs/random_read_benchmark
+```
+
+## Benchmark Methodology
+### Environment Setup
+
+#### Location Details
+- **Benchmark Machine Region**: us-west1
+- **S3 Storage Bucket Location**: ap-northeast-1
+
+#### Hardware Specifications
+
+**CPU Architecture**
```

Comment:
I prefer directly paste the config [here](https://github.com/dentiny/duck-read-cache-fs/pull/64) in code blocks